src/utils/jobs.js

```diff
@@ -0,0 +1,15 @@
+const { didTimePassed } = require("./helper")
+
+const getExcludedGuildsIds = (jobs) => {
+  const excGuilds = []
+  jobs.forEach((job) => {
+    if (!job.jobState || (job.jobInterval && !didTimePassed(job.lastJobTime, job.jobInterval))) {
+      excGuilds.push(job.guildId)
+    }
+  })
+  return excGuilds
```

Comment:
As per your reply, we should only have Guild IDs here, but we can still use ES6 logic to map:
```js
const getExcludedGuildsIds = (jobs) => {
  return jobs
    .filter((job) => !job.jobState || (job.jobInterval && !didTimePassed(job.lastJobTime, job.jobInterval)))
    .map((job) => job.guildId)
}
```

Reply:
You could even do all that in a single loop using just `.map()`

Reply:
That's more lines 😄